### PR TITLE
Submission for the java ratpack framework

### PR DESCRIPTION
**Pull request by jringuette**
Submission for the java ratpack framework

### CODE REVIEW

File: frameworks/Java/ratpack/ratpack.dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM gradle:4.7.0-jdk8 as gradle
+USER root
+WORKDIR /ratpack
+COPY build.gradle build.gradle
+COPY src src
+RUN gradle shadowJar
```
Comment (msmith-techempower) [inline frameworks/Java/ratpack/ratpack.dockerfile]: `gradle-wrapper.jar` either needs to be downloaded here when building the image, or in a dependency management script.

File: frameworks/Java/ratpack/ratpack.dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM gradle:4.7.0-jdk8 as gradle
```
Comment (msmith-techempower) [inline frameworks/Java/ratpack/ratpack.dockerfile]: Presumably, this puts `gradlew` on the path, and therefore the inclusion of that file in this codebase is unnecessary.

File: frameworks/Java/ratpack/ratpack.dockerfile (hunk 1 above)
Comment (jringuette) [reply frameworks/Java/ratpack/ratpack.dockerfile]: Done, I've removed it. It is not required for building/running via docker. It was just a convenience to build via intellij without having to install a system wide gradle.

File: frameworks/Java/ratpack/ratpack.dockerfile (hunk 2 above)
Comment (jringuette) [reply frameworks/Java/ratpack/ratpack.dockerfile]: See above

**Pull request merged**